src/App.js

```diff
@@ -1,11 +1,32 @@
-import React from 'react';
-
-function App() {
-  return (
-    <h1>
-      Hello Wors
-    </h1>
-  );
+import React, { Component } from 'react';
+import Header from './Components/Header/Header';
+import Form from './Components/Form/Form';
+import WeatherDetails from './Components/WeatherDetails/WeatherDetails';
```

Comment:
please add this package
https://www.npmjs.com/package/eslint-plugin-import
and this lines to your eslintrc 

```
 "import/first": "error",
    "import/newline-after-import": "error",
    "import/no-duplicates": "error",
    "import/no-extraneous-dependencies": ["error", { "devDependencies": true }],
    "import/no-named-default": "error",
    "import/prefer-default-export": "off",
```

Then eslint will warn you what is wrong with this code. 
There should be always one empty line between import groups from packages from node_modules and from your local imports